ducktape/tests/runner.py

```diff
@@ -13,24 +13,75 @@
 # limitations under the License.
 
 
-from ducktape.logger import Logger
 from ducktape.tests.result import TestResult, TestResults
+from ducktape.tests.session_context import TestContext
 
+import errno
 import logging
+import os
+import sys
+import traceback
 
 
-class TestRunner(Logger):
+class TestRunner(object):
     def __init__(self, session_context, test_classes, cluster):
         self.tests = test_classes
         self.cluster = cluster
         self.session_context = session_context
         self.results = TestResults(self.session_context)
-        logging.basicConfig(level=logging.INFO)
+        self.logger = session_context.logger
+
+        self.logger.debug("Instantiating %s" + self.__class__.__name__)
 
     def run_all_tests(self):
         raise NotImplementedError()
 
 
+def mkdir_p(path):
+    """mkdir -p functionality.
+    :type path: str
+    """
+    try:
+        os.makedirs(path)
+    except OSError as exc: # Python >2.5
+        if exc.errno == errno.EEXIST and os.path.isdir(path):
+            pass
+        else:
+            raise
+
+
+def get_test_case(test_class, session_context):
+    """Create test context object and instantiate test class.
+    :type test_class: ducktape.tests.test.Test.__class__
+    :type session_context: ducktape.tests.session_context.SessionContext
+    :rtype test_class
+    """
+    session_context.logger.debug(str(session_context))
```

Comment:
Should there be actual messages here, or are these just leftover from debugging?